### ISSUE TITLE
move Kibana off of control nodes

### ISSUE THREAD

**Issue by ryane** [CiscoCloud/mantl#1201]
Currently, Kibana is configured to run on control nodes. This has a couple of drawbacks: it is resource intensive, harder to scale, and does not support multiple instances (having multiple instances of Kibana talking to multiple Elasticsearch clusters). We should consider moving Kibana so that it runs within Mesos / Marathon. The Kibana framework is an option but not until it support Mesos framework authentication: https://github.com/mesos/kibana/issues/17

**Comment by siddharthist** [comment]
What are you thinking in the meantime? It doesn't look like that framework is very well maintained, a PR for auth has been open for months with no feedback. Should we work on a Marathon job?

**Comment by siddharthist** [comment]
I've got what I think is a pretty good start going here: https://github.com/CiscoCloud/mantl/tree/feature/kibana-marathon

Could you help me finish it @ryane? The Kibana dashboard isn't showing via MantlUI.

**Comment by siddharthist** [comment]
It looks like the root of most of the problems above is that the elasticsearch executor isn't getting activated. I tried with five 4GB DigitalOcean hosts, so I doubt it's a resource availability problem.

**Comment by siddharthist** [comment]
Is it necessary to have the Kibana configuration templated? [Looks like some of it can be done in the environment](https://hub.docker.com/_/kibana/) (see "How to Use This Image"), which we could manipulate via Marathon when posting the task. What variables do we change?

**Comment by siddharthist** [comment]
Somewhat relatedly, it looks like removing and restarting the ES/Kibana Marathon tasks results in no ES executors coming up. They aren't present in Mesos and there are no elasticsearch servers in `kibana.yml`:
```
# The Elasticsearch instance to use for all your queries.

```
We might want to file a separate issue for this if we can reproduce on another branch.
